core/input/input_event.cpp

```diff
@@ -552,40 +567,68 @@ bool InputEventMouseButton::action_match(const Ref<InputEvent> &p_event, bool *p
 
 String InputEventMouseButton::as_text() const {
 
-	String button_index_string = "";
+	String button_event_string = "";
+
 	switch (get_button_index()) {
 		case BUTTON_LEFT:
-			button_index_string = "BUTTON_LEFT";
+			button_event_string = "LMB";
 			break;
 		case BUTTON_RIGHT:
-			button_index_string = "BUTTON_RIGHT";
+			button_event_string = "RMB";
 			break;
 		case BUTTON_MIDDLE:
-			button_index_string = "BUTTON_MIDDLE";
+			button_event_string = "MMB";
 			break;
 		case BUTTON_WHEEL_UP:
-			button_index_string = "BUTTON_WHEEL_UP";
+			button_event_string = "MWU";
 			break;
 		case BUTTON_WHEEL_DOWN:
-			button_index_string = "BUTTON_WHEEL_DOWN";
+			button_event_string = "MWD";
 			break;
 		case BUTTON_WHEEL_LEFT:
-			button_index_string = "BUTTON_WHEEL_LEFT";
+			button_event_string = "MWL";
 			break;
 		case BUTTON_WHEEL_RIGHT:
-			button_index_string = "BUTTON_WHEEL_RIGHT";
+			button_event_string = "MWR";
 			break;
 		case BUTTON_XBUTTON1:
-			button_index_string = "BUTTON_XBUTTON1";
+			button_event_string = "MBX1";
 			break;
 		case BUTTON_XBUTTON2:
-			button_index_string = "BUTTON_XBUTTON2";
+			button_event_string = "MBX2";
 			break;
 		default:
-			button_index_string = itos(get_button_index());
+			button_event_string = itos(get_button_index());
```

Comment:
Wouldn't it make it more consistent to prefix this with "MB" or something like that? Otherwise, it's just a number, which is going to look wrong in combinations.

Reply:
I agree but I don't think MB would work since MB4, MB5 are commonly used in other games to represent what in Godot is MBX1 and MBX2, which could case some confusion. It's not particularly clean, but maybe something like `Unknown Button N`

Reply:
Actually I just realised the numbers would only start at 9 so this should be fine.